src/epivizFileParser/BamFile.py

```diff
@@ -50,10 +52,11 @@ def get_col_names(self, result):
         Columns of a bam file
         """
         if self.columns is None:
-            self.columns = ["chr", "start", "end", "number of sequence aligned"]
+            self.columns = ["chr", "start", "end",
+                            "number of sequence aligned"]
```

Comment:
lets call this value

Reply:
columns?

also that was linter doing it's things

merged already...